src/System.Net.Sockets.Legacy/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -51,6 +51,7 @@ public SendPacketsAsync(ITestOutputHelper output)
         #region Basic Arguments
 
         [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
> It would be interesting to understand why: maybe a TODO item to add Unix tests that clarify the behavior?

None of these apply to non-Windows platforms because the API underlying `SendPackets` is peculiar to Winsock. These should probably instead be excluded on non-Windows platforms with an issue to track whether or not we need to provide support for these APIs on those platforms.

> If none of the tests apply for Nix, shouldn't we only include it for Windows in the csproj?

This is debatable. In this case, no, due to the aforementioned uncertainty re: the necessity of the APIs, but perhaps in the case of Disconnect (where there is no possible implementation on other platforms) we should exclude those files.